src/renderer/renderElement.ts

```diff
@@ -334,22 +336,64 @@ const generateElementShape = (
 
         // add lines only in arrow
         if (element.type === "arrow") {
-          const arrowPoints = getArrowPoints(element, shape);
-          if (arrowPoints) {
-            const [x2, y2, x3, y3, x4, y4] = arrowPoints;
-            // for dotted arrows caps, reduce gap to make it more legible
+          const { startDecorator = null, endDecorator = "arrow" } = element;
```

Comment:
What about using enum or some sort of type for the decorators?

(Maybe we should also use that for shapes and other strings.. but that's for a separate PR)

Reply:
Not sure what you mean, these are already typed.

Note: I'd rather not use enums in general (though I'm aware we use them in one or two places already), as it's a TS feature that has many weird rules that are hard to remember. It's also introduces runtime constructs (i.e. it's not erasable so we won't ever be able to use tools which only erase types, such as esbuild). And lastly, its use is discouraged by TS maintainers themselves.

Reply:
`DECORATOR.ARROW` instead of strings.. for autocompletion and everything

Reply:
They're already typed so there's no need for adding constants on top of them, we'd end up having constants for every typed string, which would be duplicating of the effort.

Note: I'd rather not use enums in general (though I'm aware we use them in one or two places already), as it's a TS feature that has many weird rules that are hard to remember. It's also introduces runtime constructs (i.e. it's not erasable so we won't ever be able to use tools which only erase types, such as esbuild). And lastly, its use is discouraged by TS maintainers themselves.

Reply:
> What about using enum or some sort of type for the decorators?

For what it's worth, ESLint is also not set up to handle enums. (It throws an unused-var warning).